docs/jasmine-upgrade.md

```diff
@@ -0,0 +1,76 @@
+## Upgrading from Jasmine 1.3 to 2.0
+
+First, please read the official upgrade document: http://jasmine.github.io/2.0/upgrading.html
+
+#### Focused specs
+
+Instead of `iit`, please use `fit`. Instead of `ddescribe`, please use `fdescribe`.
+
+#### Custom matchers
+
+See http://jasmine.github.io/2.0/upgrading.html#section-Custom_Matchers
+
+Before:
+```javascript
+toBeCustom: function(expected) {
+  var passed = this.actual === expected;
+  this.message = function() {
+    return [
+      'Expected ' + this.actual + ' to equal ' + expected,
+      'Expected ' + this.actual + ' not to equal ' + expected
+    ];
+  };
+  return passed;
+};
+```
+
+Now:
+```javascript
+toBeCustom: function(util, customEqualityTesters) {
+  return {
+    compare: function(actual, expected) {
+      var passed = actual === expected
+      return {
+        pass: passed,
+        message: 'Expected ' + actual + (passed ? '' : ' not') + ' to equal ' + expected
+      };
+    }
+  };
+}
+```
+
+#### Asynchronous specs
+
+You can still pass in the done parameter as part of your asynchronous spec, but the syntax for failing it has changed.
+
+Before:
+```javascript
+it("async spec", function(done) {
+  setTimeout(function() {
+    if (passed) {
+      done(); // When done
+    } else {
+      done('failure message'); // To fail spec
+    }
+  }, 5000);
+});
+```
+
+Now:
+```javascript
+it("async spec", function(done) {
+  setTimeout(function() {
+    if (passed) {
+      done(); // When done
+    } else {
+      done.fail('failure message'); // To fail spec
+        // or
+      this.fail('failure message'); // Also fails spec
```

Comment:
yes good catch. I didn't actually run the example, and it probably won't work. 
I'm just going to remove the reference to `this.fail` since it feels more consistent to use `done.fail` anyway